java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/RemoteSePluginImpl.java

```diff
@@ -39,26 +37,32 @@
     public static final String DEFAULT_PLUGIN_NAME = "RemoteSePlugin";
```

Comment:
the class RemoteSePluginImpl should extend AbstractThreadedObservablePlugin instead of AbstractPlugin